frontends/p4/inlining.h

```diff
@@ -100,12 +100,67 @@ struct PerInstanceSubstitutions {
 struct InlineSummary : public IHasDbPrint {
     /// Various substitutions that must be applied for each instance
     struct PerCaller {  // information for each caller
+        /// Pair identifying all the invocations of the subparser which can use the same
```

Comment:
Please indicate that these structures are only used if that particular optimization is enabled and are unused otherwise.

Reply:
I added a note in the comment.